x-pack/legacy/plugins/snapshot_restore/public/app/services/text/text.ts

```diff
@@ -112,6 +112,31 @@ class TextService {
       },
     });
   }
+
+  public getTimeUnitLabel(timeUnit = TIME_UNITS.SECOND, timeValue = '0') {
```

Comment:
Will we ever call just `getTimeUnitLabel()` and have a label `0 seconds`? Just wondering whether the defaults are meaningful here and if we shouldn't require these values to be provided by consumers of this service.